src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -216,7 +216,7 @@ public void M(int x)
 ILocalFunctionStatement (Symbol: System.Threading.Tasks.Task<System.Int32> LocalAsync(System.Int32 p1)) (OperationKind.LocalFunctionStatement) (Syntax: 'async Task< ... }')
   IBlockStatement (2 statements) (OperationKind.BlockStatement) (Syntax: '{ ... }')
     IExpressionStatement (OperationKind.ExpressionStatement) (Syntax: 'await Task.Delay(0);')
-      Expression: IAwaitExpression (OperationKind.AwaitExpression, Type: System.Void) (Syntax: 'await Task.Delay(0)')
+      Expression: IAwaitExpression (OperationKind.None) (Syntax: 'await Task.Delay(0)')
```

Comment:
Shouldn't these be `NoneOperation`?

Reply:
They will be so for the external API users, just the internal implementation is still based off IAwaitExpression and hence the test dumps it such.

Reply:
With both of the API internal changes I made, I also removed the calls in `CreateInternal`, so it would return a NoneOperation.